aimlflow/cli.py

```diff
@@ -25,18 +25,19 @@ def cli_entry_point():
                                                             writable=True))
 @click.option('--mlflow-tracking-uri',  required=False, default=None)
 @click.option('--experiment', '-e', required=False, default=None)
-def sync(aim_repo, mlflow_tracking_uri, experiment):
+@click.option('--excluded-artifacts', required=False, default=None)
+def sync(aim_repo, mlflow_tracking_uri, experiment, excluded_artifacts):
```

Comment:
`--excluded_adtifacts` or `--exclude_adtifacts` 🤔 ?
I'm more inclined to the second one.

Reply:
BTW, what about multiple values for artifact path patterns? I think it would make sense to support those.

Reply:
Makes sense, updated 🙌